src/main/java/com/aws/greengrass/util/platforms/windows/WindowsExec.java

```diff
@@ -99,16 +137,31 @@ public synchronized void close() throws IOException {
         Process killerProcess = new ProcessBuilder().command("taskkill", "/f", "/t", "/pid",
                 Integer.toString(Processes.newPidProcess(process).getPid())).start();
```

Comment:
Michael's comment about logging commands applies here too (even if it's just at trace)